umi_tools/umi_tools.py

```diff
@@ -47,8 +47,6 @@ def main():
     elif len(argv) > 2 and  argv[2] in ["--help", "-h", "--help-extended"]:
         print("UMI-Tools: Version %s" % __version__)
```

Comment:
Could add a (hidden) option to output help without the version number to resolve comment above?

Reply:
or just only output the version number with `--version`

Reply:
Yeah, I think that would be the best option